### PR TITLE
Feature/make affiliate countries fancier

### PR DESCRIPTION
**Pull request by tuupke**
This implements the dropdown for the selection of the country for an affiliation.
So we do not need to look up the country codes anymore. Most if not all occurrences of the old ISO3166-A3 codes should be replaced with the full version.

### CODE REVIEW

**Comment by nickygerritsen** [comment]
Now I'm lost 😛 Whats the difference with #544 ? Also I think we should still split the formatting change of Util into a separate commit or is that not what @meisterT meant?

File: webapp/src/DOMJudgeBundle/Utils/Utils.php [hunk 1]
```diff
@@ -1087,10 +1120,17 @@ public static function generatePassword()
...
+            case 'm':
+                return (int)$size_str * 1048576;
+            case 'K':
+            case 'k':
+                return (int)$size_str * 1024;
+            case 'G':
+            case 'g':
+                return (int)$size_str * 1073741824;
+            default:
+                return $size_str;
```
Comment (thijskh) [inline webapp/src/DOMJudgeBundle/Utils/Utils.php]: Not cleaner to just strtolower?

File: webapp/src/DOMJudgeBundle/Utils/Utils.php [hunk 2]
```diff
@@ -1068,8 +1069,40 @@ public static function wrap_unquoted(string $text, int $width = 75, string $quot
...
+            'z',
+            '2',
+            '3',
+            '4',
+            '5',
+            '6',
+            '7',
+            '8',
+            '9'
+        ];
```
Comment (thijskh) [inline webapp/src/DOMJudgeBundle/Utils/Utils.php]: Not sure this formatting change is an improvement.

**Comment by thijskh** [comment]
I understand it.

I do think indeed that the numerous whitespace changes in Utils, although generally a good idea, should be a separate commit. We can accept a change here or there but an entire reformat of a file should be separate in the history.

**Comment by nickygerritsen** [comment]
OK I'm not lost anymore, I just thought the other PR already contained this stuff :)

File: webapp/src/DOMJudgeBundle/Resources/views/jury/partials/team_affiliation_form.html.twig [hunk 3]
```diff
@@ -7,13 +7,44 @@
         {{ form_row(form.shortname) }}
         {{ form_row(form.name) }}
         <div class="form-group">
-            {{ form_label(form.country) }}
```
Comment (thijskh) [inline webapp/src/DOMJudgeBundle/Resources/views/jury/partials/team_affiliation_form.html.twig]: Is it intentional that this label is dropped? Form looks a bit stranger now.

Comment (nickygerritsen) [reply webapp/src/DOMJudgeBundle/Resources/views/jury/partials/team_affiliation_form.html.twig]: Also I think we can now replace the whole `<div>` with just a `{{ form_row(form.country) }}` or even replace the whole form stuff with `{{ form(form) }}` since we do no custom stuff anymore

File: webapp/src/DOMJudgeBundle/Utils/Utils.php (hunk 1 above)
Comment (eldering) [reply webapp/src/DOMJudgeBundle/Utils/Utils.php]: If removing the nested case on a single line, then `strtolower` is cleaner indeed. Also, if changing this, then I'd prefer to sort by modifier size, so k,m,g (or reverse).

**Comment by tuupke** [comment]
With regards to the changes in Util, these were unintended; will revert them in a bit

File: webapp/src/DOMJudgeBundle/Resources/views/jury/partials/team_affiliation_form.html.twig (hunk 3 above)
Comment (tuupke) [reply webapp/src/DOMJudgeBundle/Resources/views/jury/partials/team_affiliation_form.html.twig]: Done!

**Comment by tuupke** [comment]
Util changes should be reverted now

File: webapp/src/DOMJudgeBundle/Utils/Utils.php (hunk 1 above)
Comment (tuupke) [reply webapp/src/DOMJudgeBundle/Utils/Utils.php]: Utils changes have been removed.

File: webapp/src/DOMJudgeBundle/Resources/views/jury/partials/team_affiliation_form.html.twig [hunk 4]
```diff
@@ -1,19 +1,38 @@
...
+            : '<img src="' + imgBase.replace("NLD.png", element.id + ".png", 1) + '" class="countryflag">&nbsp;';
+
+        return img + element.text;
+    };
+
+    $('#team_affiliation_country').select2({
+        templateResult: template,
+        templateSelection: template,
+
+        // Prevent escaping
```
Comment (meisterT) [inline webapp/src/DOMJudgeBundle/Resources/views/jury/partials/team_affiliation_form.html.twig]: why is this necessary?

Comment (nickygerritsen) [reply webapp/src/DOMJudgeBundle/Resources/views/jury/partials/team_affiliation_form.html.twig]: Otherwise you see `<img...` instead of an image
